examples/typeclass-tests.dx

```diff
@@ -0,0 +1,10 @@
+interface TestInterface where
```

Comment:
You have to add this file to the `makefile` test recipe, or else it won't be checked!